### PR TITLE
add new satellite source - esri

### PR DESCRIPTION
**Pull request by sarveshpro**

### CODE REVIEW

File: src/features/projects/components/maps/TimeTravel.tsx [hunk 1]
```diff
@@ -445,6 +445,123 @@ export default function TimeTravel({}: Props): ReactElement {
           }
         });
       }
+
+          // ESRI
+    if (selectedSource1 === 'esri') {
```
Comment (norbertschuler) [inline src/features/projects/components/maps/TimeTravel.tsx:450]: I do not see the source `esri` when looking at the Vercel deployment - how can I test it?

Bildschirmfoto 2021-08-17 um 10 55 56

Comment (norbertschuler) [reply src/features/projects/components/maps/TimeTravel.tsx:450]: I think it only appears after https://github.com/Plant-for-the-Planet-org/planet-geo-api/pull/8 got deployed, right?

Comment (sarveshpro) [reply src/features/projects/components/maps/TimeTravel.tsx:450]: Yes, test here until then - https://planet-test-sarvesh.herokuapp.com/

**Review by norbertschuler — APPROVED** [review]
I can see the ESRI maps :-)

**Comment by norbertschuler** [comment]
- [ ] I got an exception when visiting https://planet-webapp-pyql75xdp-planetapp.vercel.app/reunion-anillo-verde-de-granada-phase-i:

Bildschirmfoto 2021-08-17 um 11 18 18

**Comment by sarveshpro** [comment]
> * [x]  I got an exception when visiting https://planet-webapp-pyql75xdp-planetapp.vercel.app/reunion-anillo-verde-de-granada-phase-i:

Try now

**Comment by norbertschuler** [comment]
> > * [x]   I got an exception when visiting https://planet-webapp-pyql75xdp-planetapp.vercel.app/reunion-anillo-verde-de-granada-phase-i:
> 
> Try now

Ok, probably just caused by the 500 error in the raster API.

**Comment by norbertschuler** [comment]
- [ ] Why is time travel for https://planet-webapp-pyql75xdp-planetapp.vercel.app/reunion-anillo-verde-de-granada-phase-i is using Sentinel 2 but https://planet-webapp-pyql75xdp-planetapp.vercel.app/yucatan is using ESRI?

**Comment by sarveshpro** [comment]
> * [x]  Why is time travel for https://planet-webapp-pyql75xdp-planetapp.vercel.app/reunion-anillo-verde-de-granada-phase-i is using Sentinel 2 but https://planet-webapp-pyql75xdp-planetapp.vercel.app/yucatan is using ESRI?

try the new deployment. I fixed this

**Pull request merged**